rpcs3/Emu/Io/Buzz.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "Emu/Io/usb_device.h"
+
+class usb_device_buzz : public usb_device_emulated
+{
+	int first_controller;
+	int last_controller;
+
+public:
+	usb_device_buzz(int first_controller, int last_controller);
+	~usb_device_buzz();
+
+	void control_transfer(u8 bmRequestType, u8 bRequest, u16 wValue, u16 wIndex, u16 wLength, u32 buf_size, u8* buf, UsbTransfer* transfer) override;
```

Comment:
we use snake_case in this project.
Is there a reason why the arguments of this function don't follow our guidelines?

Reply:
Everything is copied from https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/Io/GHLtar.h but they are well known USB names. Nobody uses w_Value or w_Index

Reply:
The name of the params follow the usb standard which is why they're not snake case(ie it is fine).